src/Sulu/Bundle/SecurityBundle/Controller/RoleController.php

```diff
@@ -242,12 +244,14 @@ public function putAction(Request $request, $id)
         /** @var RoleInterface $role */
         $role = $this->roleRepository->findRoleById($id);
         $name = $request->get('name');
+        $key = $request->get('key');
```

Comment:
We should always specify where to get the value: `$request->request->get('key')`.